### PR TITLE
Increase ARM CI coverage

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by tamird** [comment]
@nvanbenschoten PTAL

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -28,27 +33,89 @@ jobs:
...
-            race_build: true
-          - go: '1.8'
-            arch: x64
-            race_build: true
+          # Race builds are pretty much busted on Go 1.4 and below on systems
+          # with newer C compilers. A number of fixes were backported to
+          # go1.4-bootstrap but not any released version. See
+          # https://github.com/golang/go/compare/go1.4.3...release-branch.go1.4.
+          #
+          # Cross-compilation became possible in go1.15 with the removal of C
```
Comment (nvanbenschoten) [inline .github/workflows/go.yml:49]: `s/go1.15/go1.5/`

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -87,8 +157,23 @@ jobs:
...
+          CGO_ENABLED: 1
+        run: |
+          # Non-host *.syso files are missing from the Go toolchains provided
+          # by setup-go. See https://github.com/actions/setup-go/issues/181.
+          curl --location --output ${{ env.GOROOT }}/src/runtime/race/race_linux_arm64.syso \
+            https://github.com/golang/go/raw/release-branch.go${{ matrix.go }}/src/runtime/race/race_linux_arm64.syso && \
+          sudo apt install gcc-aarch64-linux-gnu && \
+          CC=aarch64-linux-gnu-gcc CC_FOR_TARGET=gcc-aarch64-linux-gnu go test -c -race -o goid.race.test ./...
+      - name: 'DeleteTestRace with $$ {{ matrix.go}} for aarch64'
+        if: ${{ matrix.arch == 'aarch64' && !matrix.arm64_race_unsupported && matrix.arm64_unsupported_vma_range }}
```
Comment (nvanbenschoten) [inline .github/workflows/go.yml:173]: Why build the race binary just to delete it? You just want to see that it compiles?

**Review by nvanbenschoten — APPROVED** [review]
LGTM

File: .github/workflows/go.yml (hunk 2 above)
Comment (tamird) [reply .github/workflows/go.yml:173]: Yeah, just to see that it compiles.

**Review by nvanbenschoten — APPROVED** [review]

**Pull request merged**